source/applications/advanced/gamma_correction.py

```diff
@@ -0,0 +1,89 @@
+"""
+Capture a 2D image and apply a configurable gamma correction.
+"""
+
+import argparse
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    """Configure and take command line arguments from user
+
+    Returns:
+        Arguments from user
+
+    """
+    parser = argparse.ArgumentParser(
+        description=(
+            "Capture 2D image and apply gamma correction\n"
+            "Example:\n\t $ python gamma_correction.py 2"
+        ),
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    parser.add_argument(
+        "gamma", nargs="?", type=float, help="Gamma correction value", default="2"
+    )
+
+    return parser.parse_args()
+
+
+def adjust_gamma(image, gamma: float):
+    """Adjust gamma on image.
+
+    Args:
+        image: OpenCV image to be corrected
+        gamma: Gamma setting to apply
```

Comment:
Use the same tense - image to correct, settings to apply, or image to be corrected, settings to be applied, I don't know which is correct through.

Reply:
done, kept 'to be'